Bugsnag/Assets/Bugsnag/Editor/BugsnagEditor.cs

```diff
@@ -196,13 +205,23 @@ private void DrawAdvancedSettings(SerializedObject so, BugsnagSettingsObject set
         private void DrawSwitchOptions(SerializedObject so)
         {
             EditorGUI.indentLevel++;
+            EditorGUIUtility.labelWidth = 270;
             EditorGUILayout.PropertyField(so.FindProperty("SwitchCacheIndex"));
             EditorGUILayout.PropertyField(so.FindProperty("SwitchCacheMaxSize"));
             EditorGUILayout.PropertyField(so.FindProperty("SwitchCacheMountName"));
             EditorGUILayout.PropertyField(so.FindProperty("SwitchCacheType"));
             EditorGUI.indentLevel--;
         }
 
+        private void DrawSymbolUploadOptions(SerializedObject so)
+        {
+            EditorGUI.indentLevel++;
+            EditorGUIUtility.labelWidth = 270;
+            EditorGUILayout.PropertyField(so.FindProperty("AutoUploadSymbols"));
+            EditorGUILayout.PropertyField(so.FindProperty("BugsnagCLIExecutablePath"));
```

Comment:
Can we put a path.exists check somewhere? and/or at the point that the path is used.

Reply:
yeah this is already done in RunBugsnagCliCommand